docs/docs/legal/data-processing-agreement.md

```diff
@@ -0,0 +1,229 @@
+
+The Processor obligates itself to impose on all sub-processors, by means of a contract (or in another appropriate manner), the same data protection obligations as are imposed on it by this Annex. In particular, sufficient guarantees shall be provided that the appropriate technical and organizational measures are implemented in such a way that the processing by the sub-processor is carried out in accordance with the legal requirements. If the sub-processor fails to comply with its data protection obligations, the processor shall be liable to the customer for this as for its own conduct.
+
+## Assistance in responding to requests
+
+The Processor shall support the Customer as far as possible with suitable technical and organizational measures in fulfilling its obligation to respond to requests to exercise the data subject's rights. The parties shall agree separately on the compensation of the Processor for this.
+
+## Further support for the customer
+
+The Processor shall, taking into account the nature of the processing and the information available to it, assist the Customer in complying with its obligations in connection with the security of the processing, any notifications of personal data breaches, and any data protection impact assessments.
+
+## Deletion or destruction after termination
+
+Upon Customer&#39;s request, the Processor shall delete personal data received after the end of the agreement, unless there is a legal obligation for the Processor to store or further process such data.
+
+## Information and control rights of the customer
+
+The Processor shall provide the Customer with all information necessary to demonstrate compliance with the obligations set forth in this annex. It shall enable and contribute to audits, including inspections, carried out by the Customer or an auditor appointed by the Customer.
+
+The procedure to be followed in the event of directions that are presumed to be unlawful is governed by the section [Bound by directions](#bound-by-directions) of this Appendix.
+
+## Annex regarding security measures
+
+The Processor has taken the following organizational and technical security measures to ensure a level of protection of the Personal Data processed that is appropriate to the risk:
+
+### Pseudonymization / Encryption
+
+The following measures for pseudonymization and encryption exist:
+
+1. All communication is encrypted with TLS &gt;1.2 with PFS
+2. Critical data is stored in encrypted form
+3. Storage media that store customer data are always encrypted
+4. Passwords are irreversibly stored with a hash function (bcrypt)
+5. Data for web analytics are pseudonymized and do not contain any personal data
+
+### Ensuring certain properties of the systems and services
+
+### Confidentiality
```

Comment:
Is this heading correct?

Reply:
We could use "Information Security" instead. Was in German "Vertraulichkeit".

Reply:
It was not the name of the heading but the twice h3, this makes IMO no sense

Confidentiality is fine but should be h4